project-files/owasp-dependency-check/dependency-check-suppression.xml

```diff
@@ -24,4 +24,28 @@
     ]]></notes>
     <cve>CVE-2023-35116</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
```

Comment:
I think it works to have all theses grouped in a single <suppress> with multiple <vulnerbilityName> elements

Reply:
done!